src/internal/globalVars.ts

```diff
@@ -46,4 +46,5 @@ export class GlobalVars {
   public static remoteCameraErrorHandler: (error: remoteCamera.ErrorReason) => void;
   public static remoteCameraDeviceStateChangeHandler: (deviceStateChange: remoteCamera.DeviceState) => void;
   public static remoteCameraSessionStatusChangeHandler: (sessionStatusChange: remoteCamera.SessionStatus) => void;
+  public static userSettingsChangeHandler: (updatedSettingKey: userSettingKeys, updatedValue: any) => void;
```

Comment:
```suggestion
  public static userSettingsChangeHandler: (settingKey: userSettingKeys, value: any) => void;
```
The handler is already called a "change handler". I feel that adding the "updated" prefix to each param may be redundant.